### PR TITLE
add 50+ vendors to endor_accounts.yaml

### PR DESCRIPTION
**Pull request by kbroughton**
Adding a bunch of new vendors

### CODE REVIEW

File: vendor_accounts.yaml [hunk 1]
```diff
@@ -187,4 +187,200 @@
...
+  accounts: ['195990147830']
+- name: 'AlertLogic'
+  source: 'https://docs.alertlogic.com/prepare/iam-role-creation.htm#top'
+  accounts: ['239734009475', '733251395267', '857795874556']
+- name: 'CloudConformity'
+  source: 'https://github.com/cloudconformity/documentation-api/blob/master/Accounts.md#update-account'
+  accounts: ['717210094962']
+- name: 'Wavefront'
+  source: 'https://docs.wavefront.com/integrations_aws_metrics.html'
+  accounts: ['301213811993']
```
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml:211]: documentation changed

File: vendor_accounts.yaml [hunk 2]
```diff
@@ -187,4 +187,200 @@
...
+  accounts: ['414351767826']
+- name: 'trackit'
+  source: 'https://trackit.io/'
+  accounts: ['394125495069']
+- name: 'Presidio'
+  source: 'customer IAM policy'
+  accounts: ['118652503430']
+- name: 'BlueSentry'
+  source: 'customer IAM policy'
+  accounts: ['964273422046']
```
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml:385]: I'll make a new PR tonight or tomorrow with these modifications.

File: vendor_accounts.yaml [hunk 3]
```diff
@@ -187,4 +187,200 @@
...
+  accounts: ['765628985471']
+- name: 'site24x7'
+  source: 'https://www.site24x7.com/help/api/#generate-aws-external-id'
+  accounts: ['949777495771']
+- name: 'ylastic'
+  source: 'https://blog.ylastic.com/post/153943542476/iam-role-with-external-id-for-cross-account-access'
+  accounts: ['710193521658']
+- name: 'qubole'
+  source: 'https://docs.qubole.com/en/latest/user-guide/managing-accounts/aws/iam-roles/override-iam-roles.html'
+  accounts: ['8053', '805246085872']
```
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml:292]: removed 8053.

File: vendor_accounts.yaml [hunk 4]
```diff
@@ -187,4 +187,200 @@
   accounts: ['218546966473']
 - name: 'Emnify'
   source: 'https://www.emnify.com/datastreamer-integration-into-aws'
-  accounts: ['884047677700']
\ No newline at end of file
+  accounts: ['884047677700']
+- name: 'Qualys Cloud View'
+  source: 'https://qualysguard.qualys.com/qwebhelp/fo_portal/scans/ec2_connector.htm'
+  accounts: ['80595016317', '205767712438']
```
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml:193]: yes fixed

File: vendor_accounts.yaml [hunk 5]
```diff
@@ -187,4 +187,200 @@
...
+- name: 'Auth0'
+  source: 'https://auth0.com/docs/integrations/aws/sso'
+  accounts: ['951887872838']
+- name: 'Altus'
+  source: 'https://docs.cloudera.com/documentation/altus/topics/alt_env_setup_aws.html#env_quickstart'
+  accounts: ['062663104137']
+- name: 'Cloud Optix'
+  source: 'https://community.sophos.com/kb/en-us/126082'
+  accounts: ['195990147830']
```
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml:202]: only available with free account. Removed

File: vendor_accounts.yaml [hunk 6]
```diff
@@ -187,4 +187,200 @@
...
+  accounts: ['759994998399']
+- name: 'ThingSpace'
+  source: 'https://thingspace.verizon.com/resources/documentation/cloudconnector/Getting_Started/Streaming_to_AWS/'
+  accounts: ['675479154635']
+- name: 'Anodot'
+  source: 'https://support.anodot.com/hc/en-us/articles/360012563774'
+  accounts: ['340481513670']
+- name: 'CDM Solutions'
+  source: 'https://pastebin.com/HK3STK0v'
+  accounts: ['445937625421']
```
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml:370]: removed

File: vendor_accounts.yaml [hunk 7]
```diff
@@ -187,4 +187,200 @@
...
+- name: 'MediaMath'
+  source: 'https://apidocs.mediamath.com/reporting/log-level-data-service/overview#data-security-and-authorization'
+  accounts: ['794878508631']
+- name: 'https://docs.databricks.com/administration-guide/account-settings/aws-accounts.html,https://docs.databricks.com/administration-guide/account-settings/aws-storage.html'
+  source: 'https://docs.databricks.com/administration-guide/account-settings/aws-accounts.html'
+  accounts: ['414351767826']
+- name: 'trackit'
+  source: 'https://trackit.io/'
+  accounts: ['394125495069']
```
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml:379]: also from pastebin. removed

File: vendor_accounts.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml:385]: removed

File: vendor_accounts.yaml [hunk 8]
```diff
@@ -187,4 +187,200 @@
...
+- name: 'Presidio'
+  source: 'customer IAM policy'
+  accounts: ['118652503430']
```
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml:382]: removed

File: vendor_accounts.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml:208]: This was publicly available but it isn't indexed in wayback machine, so removing.

File: vendor_accounts.yaml [hunk 9]
```diff
@@ -187,4 +187,200 @@
...
+- name: 'Anodot'
+  source: 'https://support.anodot.com/hc/en-us/articles/360012563774'
+  accounts: ['340481513670']
+- name: 'CDM Solutions'
+  source: 'https://pastebin.com/HK3STK0v'
+  accounts: ['445937625421']
+- name: 'MediaMath'
+  source: 'https://apidocs.mediamath.com/reporting/log-level-data-service/overview#data-security-and-authorization'
+  accounts: ['794878508631']
+- name: 'https://docs.databricks.com/administration-guide/account-settings/aws-accounts.html,https://docs.databricks.com/administration-guide/account-settings/aws-storage.html'
```
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml]: fixed

File: vendor_accounts.yaml [hunk 10]
```diff
@@ -187,4 +187,200 @@
...
+  accounts: ['884047677700']
+- name: 'Qualys Cloud View'
+  source: 'https://qualysguard.qualys.com/qwebhelp/fo_portal/scans/ec2_connector.htm'
+  accounts: ['80595016317', '205767712438']
+- name: 'Auth0'
+  source: 'https://auth0.com/docs/integrations/aws/sso'
+  accounts: ['951887872838']
+- name: 'Altus'
+  source: 'https://docs.cloudera.com/documentation/altus/topics/alt_env_setup_aws.html#env_quickstart'
+  accounts: ['062663104137']
```
[the remark this answers is not in the document]
Comment (kbroughton) [reply vendor_accounts.yaml:199]: This one is available on their blog https://blog.cloudera.com/cloudera-altus-cloud-services-youre-in-control-part-2/
though they removed it from the quickstart. Changed the source ref.

**Comment by kbroughton** [comment]
Just found that Security Center has published a bunch of AWS account id's for integrations. I know at least some of them, like Crowdstrike, had made it very difficult to get their account # (non-public). https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub.pdf

**Comment by 0xdabbad00** [comment]
@kbroughton Thank you and I'm sorry I haven't gotten to this. I'm going to get this merged this week.

**Comment by 0xdabbad00** [comment]
I've made the needed adjustments by resubmitting this as #820. You will still show up in the commit history @kbroughton, but I need to close this one so the corrected one can get merged.

**Pull request closed without merging**